src/node/internal/compatibility-flags.d.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+// Keep this in sync with compatibility-date.capnp
+// TODO(soon): See if we can automatically generate this from the capnp file
```

Comment:
Is there any .d.ts generation for capnproto already? Maybe there's a typescript support. We could leverage that here.

Reply:
I really don't know.